src/ziggurat/config.clj

```diff
@@ -95,6 +96,9 @@
 (defn ssl-config []
   (get-in config [:ziggurat :ssl]))
 
+(defn sasl-config []
+  (get-in config [:ziggurat :sasl]))
```

Comment:
The problem here with this approach of having a single sasl config is that we cannot have seperate ACL for each streams. 
This needs to be implemented. The acl should be per stream as each stream can have different topics its reading from hence having separate ACL.

Reply:
-  As discussed in the thread we are currently okay with keeping actor-level changes.
- To support per-stream changes, (when required) the following plan should be done:
1. Update `streams-config-mapping-table` to start setting ACL properties as well. (Currently it only picks up auto-offset reset configs)
2. Update the `add-sasl-properties` to putIfAbsent with actor level ACL if stream ACL is missing. [updating this in next commit]